CONTRIBUTING.md

```diff
@@ -24,8 +24,11 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Write tests for your feature, or regression tests highlighting a bug
 4. Write the feature itself, or fix your bug
-5. Commit your changes (`git commit -am 'Add some feature'`)
-6. Push to the branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
+5. Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
+with a brief description of any breaking changes, fixes, features, or
+miscellaneous changes under the proper version section.
+6. Commit your changes (`git commit -am 'Add some feature'`)
+7. Push to the branch (`git push origin my-new-feature`)
+8. Create a new Pull Request
 
 Remember to squash your commits and rebase off `master`.
```

Comment:
oh, and also, at no point should master be *merged* in to the feature branch.  That's what messed me up early on.